Dockerfile-grocy

```diff
@@ -1,6 +1,8 @@
 FROM php:7.2-fpm-alpine
 LABEL maintainer="Talmai Oliveira <to@talm.ai>"
 
+ARG GROCY_VERSION=v2.6.1
```

Comment:
@berrnd @talmai Hopefully this is a good all-round approach:

- The grocy version number is specified clearly within the repository contents
- It's easy to override at image-build-time for development & testing purposes
- It doesn't require using fixed file names in GitHub release assets

Wdyr?